supervision/dataset/formats/coco.py

```diff
@@ -159,16 +173,29 @@ def detections_to_coco_annotations(
     return coco_annotations, annotation_id
 
 
+def get_coco_class_index_mapping(annotations_path: str) -> Dict[int, int]:
```

Comment:
This function will become part of the public supervision API.

- Just like with box_iou_batch_with_jaccard, we need to IMPORT it in `supervision/__init__.py` and include it in `__all__`.

- We also need a docstring that explains the purpose of this function, the use cases where it would be useful, and how it works.

Reply:
:heavy_check_mark: Done!